aiohttp/payload.py

```diff
@@ -307,17 +307,36 @@ def __init__(
             if hdrs.CONTENT_DISPOSITION not in self.headers:
                 self.set_content_disposition(disposition, filename=self._filename)
 
+        self._writable = True
+
+        try:
+            self._seekable = self._value.seekable()
```

Comment:
I think `seekable` can be blocking , at least `aiofiles` delegates it the executor https://pypi.org/project/aiofiles/